bin/alluxio-start.sh

```diff
@@ -184,12 +201,38 @@ done
 shift $((OPTIND-1))
```

Comment:
following google shell style guide, it would be great to put the script "body" into a main function: https://google.github.io/styleguide/shell.xml